app/routes/docs.contributing.github.mdx

```diff
@@ -0,0 +1,49 @@
+# GitHub
+
+For all projects in the [nasa-gcn](https://github.com/nasa-gcn) GitHub organization, contributors should adhere to the following steps:
+
+- Fork a copy of our repo
+
+- Clone your repo to your machine
+
+- Create a branch for your new changes
+
+- Create a Pull Request to contribute to the original project
+
+For more information, check out our [web-dev-bootcamp](https://github.com/nasa-gcn/web-dev-bootcamp) repository. In the syllabus folder you will find some instructive lectures and examples to help understand our workflow.
+
+## Forking a Repo
+
+Navigate to the page repository page for whichever project you wish to contribute to, for example: [gcn.nasa.gov](https://github.com/nasa-gcn/gcn.nasa.gov)
+
+Click the Fork button
+
+Click Create Fork to create a new repo in your account
+
+## Cloning a Repo
+
+With your new fork now created, click the Code button and copy the URL.
+
+Open a terminal window and run the following
+
+```sh
+git clone <your copied repo url>
+```
+
+## Branching off main
+
+Within your cloned repo on your local machine, run the following to create and switch to a new branch for your code changes:
+
+```sh
+git checkout -b <your branch name>
+```
+
+## Pull Requests
+
+Once you have completed your changes on your branch, create a [pull request](https://github.com/nasa-gcn/gcn.nasa.gov/pulls).
+
```

Comment:
should we add how to do that? something brief like this:
To create a pull request, navigate to your fork in your own github repo. Click the "branches" link in the upper left hand secondary nav menu. Once you are on the branches index, click the "New pull request" button on the right hand side of the branch you would like to use. Once you have filled in your pull request title and description, click the "Create pull request" button on the lower left hand side to create the pull request.